content/docs/server_setup/server_software.mdx

```diff
@@ -0,0 +1,222 @@
+
+[MultiPaper](https://github.com/MultiPaper/MultiPaper) is a 1.20.1 fork of [Purpur](#purpur) that splits your world(s) into multiple servers, running independently.\
+Each world is split into regions (similar to Folia), and each region runs on its own server.\
+This server software is often known as "sharding", where each server is a "shard".\
+Each server communicates with each other and is able to transfer players between servers, allowing for a seamless experience.
+
+<Callout type="warn" title="Warning">
+    **MultiPaper is highly unstable!**\
+    You must consider whether you need to use it. MultiPaper is notorious for being unstable and is not recommended for production use.\
+    Although many public servers such as DonutSMP use a sharding solution, they are often custom-made and are not available to the public.
+
+    Consider using / testing other (more stable) softwares such as [Pufferfish](#pufferfish-recommended) or [Purpur](#purpur) beforehand.
+    **Ensure you thoroughly test every feature, command and mechanic in your server if you decide to use this software.**
+</Callout>
+
+<Card href="https://multipaper.io/download.html" title="Download">
+  Download a jarfile for MultiPaper here.
+</Card>
+
+## 1.8 Softwares
+There are some spigot or paper forks that are actively maintained for 1.8.x.\
+We would recommend against using 1.8 or arguably any version below latest, due to the presence of security vulnerabilities or other exploits.\
+One of the main reasons for choosing to run a 1.8.x fork is for good performance on older hardware, or for pre-1.9 PvP.\
+See below for some 1.8.x forks that are actively maintained:
+
+<Cards>
+    <Card href="https://github.com/hpfxd/PandaSpigot" title = "PandaSpigot"></Card>
+    <Card href="https://github.com/Wind-Development/WindSpigot" title = "WindSpigot"></Card>
+    <Card href="https://github.com/CobbleSword/NachoSpigot" title = "NachoSpigot"></Card>
+</Cards>
+
+## Forge
+Forge is a modded server (and API) that allows for the use of mods on your server, which can add new content, mechanics and features to the game (Unlike Vanilla, Spigot, Paper, etc).\
+Unlike Paper, it contains **no optimizations** and is often very laggy when on larger modpacks.
+
+<Card href="https://files.minecraftforge.net/net/minecraftforge/forge/" title="Download">
+  Download a copy of Forge via [minecraftforge.net](https://files.minecraftforge.net/net/minecraftforge/forge/).
+</Card>
+
+## NeoForge
+NeoForge is a fork of Forge, maintained by most of the original Forge development team - excluding LexManos (see [this post](https://neoforged.net/news/theproject/)).\
+It aims to fix the issues surrounding the community and provide a better development experience for modders and more recent versions of Minecraft itself.
+
+<Callout type="info" title="Note">
+  Due to NeoForge being a relatively new server software, there is not as many mods that completely support it compared to the likes of Forge and Fabric.
```

Comment:
> there is not as many mods

there **are** not as many mods

Reply:
Resolved in [3552391](https://github.com/admincraft-community/admincraftwiki/pull/1/commits/35523917b01154831665f92765e0f2c48794f280)